src/components/MeasurementOverlay/MeasurementOverlay.scss

```diff
@@ -15,7 +15,6 @@
     right: 8px;
     padding: 8px;
     font-size: 14px;
-    z-index: 50;
```

Comment:
@khein-pdftron as discussed. removed this so that z-index is same for both mobile and non-mobile